nano/node/voting.hpp

```diff
@@ -122,6 +151,8 @@ class vote_generator final
 	bool started{ false };
 	std::thread thread;
 
+	friend class vote_generator_vote_spacing_Test;
+	friend class vote_generator_vote_rapid_Test;
```

Comment:
There are 2 new `vote_spacing` tests added but it doesn't match these. So it doesn't look like these friend declarations are necessary anymore (and also the nano namespace around the tests)